test/rules_std_test.py

```diff
@@ -225,6 +225,8 @@ def assert_rule_pass_in_sql(code, sql, configs=None):
     ('L032', 'pass', 'select x.a from x inner join y on x.id = y.id', None, None),
     ('L032', 'fail', 'select x.a from x inner join y using (id)', None, None),
     ('L032', 'fail', 'select x.a from x inner join y on x.id = y.id inner join z using (id)', None, None),
+    ('L033', 'pass', 'SELECT a, b FROM tbl UNION ALL SELECT c, d FROM tbl1', None, None),
+    ('L033', 'fail', 'SELECT a, b FROM tbl UNION SELECT c, d FROM tbl1', None, None),
```

Comment:
I love these test cases, could you add two more which cover the edge cases I mention above.?

1. Checking that similar queries pass/fail correctly with newlines and commends in between the `UNION` and `SELECT`.
2. Checking that `UNION DISTINCT` passes (even though there is no `ALL` in the statement).